composer_local_dev/docker_files/entrypoint.sh

```diff
@@ -28,6 +29,15 @@ if [ -f /var/local/setup_python_command.sh ]; then
     /var/local/setup_python_command.sh
 fi
 
+
+pip3 install keyring keyrings.google-artifactregistry-auth
+if [ -z "${PRIVATE_PIP_INDEX}" ] 
+then
+  echo "no private index set"
+else
+  echo "index is set to ${PRIVATE_PIP_INDEX}"
```

Comment:
Please change to: `Adding private PyPI repository index: ${PIP_EXTRA_INDEX_URL}`

Reply:
My comments seem to have disappeared but this is addressed within [changes to this commit](https://github.com/GoogleCloudPlatform/composer-local-dev/pull/21/commits/45a90d27ffc3b7d554c11924d0d91bb00eeb3872)

Reply:
Extra index URLs are unsafe. Would you guys be interested in adding support for PIP_INDEX_URL as well? Thank you